ASV44/ChatMessageBroker
# Add websocket communication protocol support

### Background

Previously broker supports only raw tcp socket connection, but currently more modern clients are oriented to support websocket communication protocol. Websocket protocol is based on HTTP and offers easier communication mechanism and easier integration on client side.

### What has been done?
1. Add broker websocket protocol communication support
2. Update broker to provide connection for both tcp sockets and websockets

### How to test?
1. Open client app and check tcp communication protocol.
2. Find websocket client and connect to broker to check of websocket communication works properly

Currently client app from this repo does not support websocket communication. It is planned to add websocket integration to 2 samples project on Android and iOS to check websocket communication;

### Checklist
